pdns/syncres.hh

```diff
@@ -736,6 +736,19 @@ public:
     d_queryReceivedOverTCP = tcp;
   }
 
+  static bool isUnsupported(QType qtype)
+  {
+    switch (qtype.getCode()) {
+      // Internal types
+    case QType::ENT:
+    case QType::ADDR:
+    case QType::ALIAS:
+    case QType::LUA:
```

Comment:
Other than "looks weird in logs", there is no reason to treat ALIAS and LUA different in the Recursor. How about not loading those two types at all?

Reply:
Well, somebody could still specify a query with that qtype, type being loaded or not.